EngineBase.py

```diff
@@ -74,11 +74,15 @@ def get_epoch_model(cls, config):
 
     load_model_epoch_filename = config.value('load', '')
     if load_model_epoch_filename:
-      assert os.path.exists(load_model_epoch_filename + get_model_filename_postfix())
+      if not load_model_epoch_filename.endswith(get_model_filename_postfix()):
+        load_model_epoch_filename = load_model_epoch_filename + get_model_filename_postfix()
+      assert os.path.exists(load_model_epoch_filename)
```

Comment:
The variable `load_model_epoch_filename` should **not** contain the filename postfix. That variable might get returned by this function `get_epoch_model`, and that is being used for multiple purpose, in many places. And many other code expects that it gets the model name there (as path, but **excluding** any filename postfix).

So, better would be to just remove the postfix in case it is there.